src/System.Security.Cryptography.Cng/ref/System.Security.Cryptography.Cng.csproj

```diff
@@ -2,10 +2,16 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
+    <AssemblyVersion>4.3.0.0</AssemblyVersion>
```

Comment:
Instead of adding a default assemblyversion here,  could you remove this line from here (and from your src project as well) and update the dir.props bellow src/System.Security.Cryptography.Cng instead?